packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -196,13 +248,7 @@ TODO:
                 source: sourceId,
             });
 
-            map.off('mousemove', layerId, addTooltip);
-            map.off('mouseleave', layerId, removeTooltip);
-
-            if (renderTooltip) {
-                map.on('mousemove', layerId, addTooltip);
-                map.on('mouseleave', layerId, removeTooltip);
-            }
+            handleInteractivity(interactive);
```

Comment:
```suggestion
            …
            
    $: if (mapReady) {
        handleInteractivity(interactive);
    }
```
I think it separates the functions a bit more and is more explicit on what trigger the reactive statement (although it is reactive on `interactive` already).

Reply:
Yes ! Let separate them 👍